pkgs/games/mindustry/default.nix

```diff
@@ -155,11 +164,25 @@ stdenv.mkDerivation rec {
 
     pushd Mindustry
   '' + optionalString enableClient ''
+    glewlib=${glew.out}/lib/libGLEW.so
+    sdllib=${SDL2}/lib/libSDL2.so
+    if ! test -f "$glewlib"; then
+      echo "libGLEW not found"
+      exit 1
+    else
+      echo "libGLEW found"
+    fi
+    if ! test -f "$sdllib"; then
+      echo "libSDL2 not found"
+      exit 1
+    else
+      echo "libSDL2 found"
+    fi
```

Comment:
We don't need any of the checking code if we use ``lib.getLib glew``.

Reply:
Thanks, that should be a nicer solution.

Reply:
`lib.getLib` only returns the path that contains the libraries, still not checking if the _wanted_ library is available.

Reply:
I did add nicer logging and added usage of the `lib.getLib` functionality. Still think it's nice to validate that the files actually exist.

Reply:
Why shouldn't the files exist? They are fundamental to the libraries. I doubt they get ever removed.

Reply:
I removed the checks.